src/crimson/osd/objclass.cc

```diff
@@ -553,7 +553,7 @@ uint64_t cls_get_osd_min_alloc_size(cls_method_context_t hctx) {
 }
 
 int cls_cxx_gather(cls_method_context_t hctx, const std::set<std::string> &src_objs, const std::string& pool,
-		   const char *cls, const char *method, bufferlist& inbl)
+		   const char *cls, const char *method, bufferlist& inbl) __attribute__ ((deprecated))
```

Comment:
ACK.

Reply:
@rzarzynski missed that, i completely removed that from crimson, and also removed the test since it is deprecated